content/files/Akeneo PIM API.postman_collection.json

```diff
@@ -706,6 +782,56 @@
                 "rest",
                 "v1",
                 "categories"
```

Comment:
There are missing query parameters on this endpoint GET /api/rest/v1/categories/{code}
- with_position

Reply:
Same  question here, isn't it https://github.com/akeneo/pim-api-docs/pull/599/files#diff-965e2072337a3d3d1d8e0ba37d6d642f21cbaf9d9c009c75a6fa016b25516cc3R830 ?

Reply:
Yes it is this option but it is also available for the GET of a category